drupal/.gitignore

```diff
@@ -24,3 +24,22 @@ web/_ping.php
 
 # Ignore files generated by PhpStorm
 /.idea/
+
+# Ignore files generated by scaffold
+web/.csslintrc
+web/.editorconfig
+web/.eslintignore
+web/.eslintrc.json
+web/.gitattributes
+web/.htaccess
+web/autoload.php
+web/index.php
+web/sites/default/default.services.yml
+web/sites/default/default.settings.php
+web/sites/development.services.yml
+web/sites/example.settings.local.php
+web/sites/example.sites.php
+web/update.php
+web/web.config
+web/_ping.php
+web/robots.txt%
```

Comment:
Is the scaffold always overwriting robots.txt. Otherwise robots.txt should be included in the repo.

Reply:
Yeah, scaffold will always overwrite robots.txt by default. So if a project requires a custom robots.txt, it needs to add this the composer.json in extra: {}

```        
"drupal-scaffold": {
   "excludes": [
      "robots.txt"
   ]
}
```